extension/src/paymentHandler/payment-handler.js

```diff
@@ -77,6 +78,11 @@ function _getPaymentHandlers(paymentObject) {
     !paymentObject.custom.fields.getPaymentMethodsResponse
   )
     handlers.push(getPaymentMethodsHandler)
+  if (
+    paymentObject.custom.fields.getCarbonOffsetCostsRequest &&
+    !paymentObject.custom.fields.getCarbonOffsetCostsResponse
```

Comment:
I wonder if we should validate if request is done after makePaymentResponse - which does not make much sense anymore?

Reply:
It's same for getPaymentMethodsRequest. We could put additional restriction here but I don't know if the integration users writes implementations with undocumented patterns 🤔 

@butenkor @lojzatran what do you think should I add a restriction for those that those could be used only before makePayment.